config/database.yml-example

```diff
@@ -3,7 +3,6 @@
 #
 default: &default
   adapter: postgresql
-  encoding: unicode
```

Comment:
Going to want to reword this commit when you squash down

Reply:
Anymore than I did for commit message on the squash?

Reply:
Ah soz I missed that you'd already done so! Just noted the original title and added reminder.